content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -114,3 +116,151 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
 If `PersistentVolumeLabel` does not support automatic labeling of your PersistentVolumes, you should consider
 adding the labels manually (or adding support for `PersistentVolumeLabel`). With `PersistentVolumeLabel`, the scheduler prevents Pods from mounting volumes in a different zone. If your infrastructure doesn't have this constraint, you don't need to add the zone labels to the volumes at all.
 
+## node.kubernetes.io/windows-build {#nodekubernetesiowindows-build}
+
+Example: `node.kubernetes.io/windows-build=10.0.17763`
+
+Used on: Node
+
+Starting in v1.17, this label is automatically added to adapt to the situation that the same cluster uses multiple versions of windows server.
+
+The label's value is in the format "MajorVersion.MinorVersion.BuildNumber".
+
+See [node.kubernetes.io/windows-build](#nodekubernetesiowindows-build).
+
+## service.kubernetes.io/headless {#servicekubernetesioheadless}
+
+Example: `service.kubernetes.io/headless=""`
+
+Used on: Service
+
+The label is added by Controller to an Endpoint denoting if its parent Service is Headless.
+
+See [service.kubernetes.io/headless](#servicekubernetesioheadless).
+
+## kubernetes.io/service-name {#kubernetesioservice-name}
+
+Example: `kubernetes.io/service-name="nginx"`
+
+Used on: Service
+
+The label is the tag name we use to differentiate multiple services. Used currently for ELBs only.
```

Comment:
```suggestion
The label is the tag name we use to differentiate multiple Services. Used currently for ELBs only.
```

Reply:
I'd write:
```suggestion
Kubernetes uses this label to differentiate multiple Services. Used currently for ELBs only.
```
however:
I also recommend explaining ELB, or rephrasing so that this text is not needed.